# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libprelude/libprelude-no-gnutls_extra.h.patch
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

remove gnutls/extra.h as it is not available in current gnutls 3.2.x

--- a/prelude-admin/server.c	2014-04-02 02:28:40.612810486 +0200
+++ b/prelude-admin/server.c	2014-04-02 02:29:27.593895175 +0200
@@ -39,7 +39,6 @@
 
 #include <gcrypt.h>
 #include <gnutls/gnutls.h>
-#include <gnutls/extra.h>
 
 #include "prelude-client.h"
 #include "prelude-error.h"
--- a/prelude-admin/prelude-admin.c	2014-04-02 02:30:02.218687508 +0200
+++ b/prelude-admin/prelude-admin.c	2014-04-02 02:30:15.362990716 +0200
@@ -57,7 +57,6 @@
 #include <errno.h>
 #include <gnutls/gnutls.h>
 #include <gnutls/x509.h>
-#include <gnutls/extra.h>
 
 #include "common.h"
 #include "config-engine.h"
